pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.ejs

```diff
@@ -0,0 +1,226 @@
+                    <button type="button" class="btn btn-sm js-rubric-item-delete text-danger">
+                      <i class="fas fa-trash"></i>
+                      <span class="sr-only">Delete</span>
+                    </button>
+                  </td>
+                  <td style="max-width: 4rem">
+                    <input type="number" class="form-control js-rubric-item-points" step="any" required>
+                  </td>
+                  <td>
+                    <input type="text" class="form-control js-rubric-item-short-text" required maxlength="100">
+                  </td>
+                  <td>
+                    <label class="js-rubric-item-description">
+                      <button type="button" class="btn btn-sm js-rubric-item-description-field">
+                        <i class="fas fa-pencil"></i>
+                      </button>
+                    </label>
+                  </td>
+                  <td>
+                    <label class="js-rubric-item-staff-instructions">
+                      <button type="button" class="btn btn-sm js-rubric-item-description-field">
+                        <i class="fas fa-pencil"></i>
+                      </button>
+                    </label>
+                  </td>
+                  <td>
+                    New
+                  </td>
+                </tr>
+              </template>
+              <% if (variant.params || variant.true_answer || submission?.submitted_answer) { %>
+              <div class="small form-text text-muted">
+                Rubric items may be populated with the following values (click to copy):
+                <ul style="max-height: 7rem; overflow-y: auto;">
+                  <% [
+                    ...Object.keys(variant.params || {}).map((key) => ['params', key]),
+                    ...Object.keys(variant.true_answer || {}).map((key) => ['correct_answers', key]),
+                    ...Object.keys(submission?.submitted_answer || {}).map((key) => ['submitted_answers', key]),
+                  ].forEach(([group, key]) => { %>
+                    <li><code class="js-copy-on-click"
+                              data-clipboard-text="{{<%= group %>.<%= key %>}}"
+                        >{{<%= group %>.<%= key %>}}</code></li>
+                  <% }); %>
+                </ul>
+              </div>
+              <% } %>
+            </div>
+            <hr/>
+            <div class="form-check">
+              <label class="form-check-label">
+                <input class="form-check-input" type="checkbox" value="true"
+                       name="tag_for_manual_grading">
+                Require all graded submissions to be manually graded/reviewed
+              </label>
+            </div>
+          </div>
+        </div>
+        <div class="modal-footer">
+          <% if (authz_data.has_course_instance_permission_edit) { %>
+          <button type="submit" class="btn btn-primary">Confirm</button>
```

Comment:
Sometimes (I can't spot an exact pattern) when I click "Confirm" I get JS console errors:
<img width="1727" alt="image" src="https://user-images.githubusercontent.com/875833/232246063-63587eaa-71f5-4a8d-99c3-9954726441f4.png">

Reply:
I fixed one particular case where something similar could happen, which would be when the grading job conflict modal is dismissed before opening the rubric settings modal. Not sure if this is the behaviour you're seeing?

Reply:
Please let me know if you're still seeing these issues.

Reply:
This has been resolved in 6cef3e858. Issue was CSRF tokens, which were handled in a previous version, but the switch from jQuery to vanilla JS broke this.